include/irr/asset/bawformat/blobs/MeshBufferBlob.h

```diff
@@ -32,22 +37,18 @@ struct IRR_FORCE_EBO MeshBufferBlobV0 : TypedBlob<MeshBufferBlobV0, ICPUMeshBuff
 	uint32_t baseInstance;
 	uint32_t primitiveType;
 	uint32_t posAttrId;
+	bool isRightHandedCoordinateSystem;
```

Comment:
don't use `bool` they have undefined size across platforms, uint8_t is more appropriate.

Reply:
I haven't known it yet though, to fix